src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests/NetCoreApp3CompatModeTests.cs

```diff
@@ -22,6 +22,37 @@ public NetCoreApp3CompatModeTests(SingleFileSharedState fixture)
             sharedTestState = fixture;
         }
 
+        [InlineData("./extract")]
+        [InlineData("../extract")]
+        [InlineData("extract")]
+        [InlineData("extract\\foo")]
+        [Theory]
+        private void Bundle_Extraction_To_Relative_Path(string relativePath)
+        {
+            var fixture = sharedTestState.TestFixture.Copy();
+            UseSingleFileSelfContainedHost(fixture);
+            Bundler bundler = BundleHelper.BundleApp(fixture, out string singleFile, BundleOptions.BundleAllContent);
+
+            var currentDir = Directory.GetCurrentDirectory();
+            var cmd = Command.Create(singleFile);
+            Directory.SetCurrentDirectory(Path.GetDirectoryName(singleFile));
```

Comment:
We should set the working directory for the command being executed rather than for the test process.